ddtrace/contrib/celery/__init__.py

```diff
@@ -1,14 +1,11 @@
 """
-Supported versions:
```

Comment:
we can add these somewhere in the docs; we have to do this for all current instrumentation (will take care in a separated PR)